diff --git a/Android.bp b/Android.bp
index 15bae6d..2800012 100644
--- a/Android.bp
+++ b/Android.bp
@@ -107,6 +107,7 @@ rust_test_host {
     test_options: {
         unit_test: true,
     },
+    data: ["tests/documents/*"],
 }
 
 rust_test {
@@ -114,6 +115,7 @@ rust_test {
     defaults: ["xml-rs_test_defaults_xml_rs"],
     // has rustc warnings
     srcs: ["tests/event_writer.rs"],
+    data: ["tests/documents/*"],
 }
 
 rust_test_host {
